src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -16,395 +22,258 @@ servers:
-      operationId: colorize
-      # In this case we would probably want to tweak the generated code for Content-Disposition so that the
-      # delivered image is displayed inline when using the Swagger UI. Inspect the generated code for how to do this.
-      x-streamingOutput: true
-      requestBody:
-        content:
-          multipart/form-data:
-            schema:
-              type: object
-              required:
-                - image
-              properties:
-                # The POSTed image can be retrieved using imageDetail.getDataHandler().getInputStream()
-                image:
-                  description: 'The image to use as source for the colorization'
-                  type: string
-                  format: binary
-                method:
-                  description: 'The algorithm used to colorize the image'
-                  type: string
-                  enum: [Random, CNN-1, GAN-1]
-                  default: GAN-1
-                  example: GAN-1
-                intensity:
-                  description: 'The intensity of the colorization'
-                  type: number
-                  format: double
-                  # Note: Constraints are not enforced by Swagger. They are only used as a contract
-                  minimum: 0.0
-                  maximum: 1.0
-                  default: 0.8
-                  example: 0.8
-
-      responses:
-        '200':
-          description: 'The colorized image'
-          content:
-            image/jpeg:
-              schema:
-                type: string
-
-  # The ping service should be in all projects, should not do any advanced processing
-  # and should respond quickly with a simple message, e.g. "pong".
-  #
-  # Note that it is placed under /monitor. The top namespace /monitor is intended for
-  # proxying to monitoring software or manual health checks. All endpoints under that
-  # namespace should be safe to expose: No changes to state, no confidential information!
-  #
-  # Note that the more detailed /monitor/status is defined below.
-  /monitor/ping:
-    get:
-      tags:
-        - Service
-      summary: 'Ping the server to check if the server is reachable.'
-      operationId: ping
-      x-useGenericResponse: false # If enabled the method will return Response
-      x-streamingOutput: false # If enabled the method will return StreamingOutput.
-      responses:
-        '200':
-          description: 'OK'
-          content:
-            text/plain:
-              schema:
-                type: string
-        '406':
-          description: 'Not Acceptable'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-        '500':
-          description: 'Internal Error'
-          content:
-            text/plain:
-              schema:
-                type: string
+          examples:
+            Rotate 180 degrees:
+              value: "180"
```

Comment:
The first example is automatically added to the request through the OpenAPI GUI, so it should probably be `0` so users are not confused by seeing an upside-down image when the try the service.

Reply:
I don't understand why Github does not mark this as outdated, The zero rotation example has been added.

Reply:
Looks like a bug in GitHub's system. The example is indeed added.